tracer_test.go

```diff
@@ -1,46 +1,106 @@
 package otelpgx
 
-import "testing"
+import (
+	"strings"
+	"testing"
+)
 
 func TestSqlOperationName(t *testing.T) {
 	tests := []struct {
-		name    string
-		query   string
-		expName string
+		name         string
+		query        string
+		spanNameFunc func(string) string
+		expName      string
 	}{
 		{
-			name:    "Spaces only",
-			query:   "SELECT * FROM users",
-			expName: "SELECT",
+			name:         "Spaces only",
+			query:        "SELECT * FROM users",
+			spanNameFunc: nil,
+			expName:      "SELECT",
 		},
 		{
-			name:    "Newline and tab",
-			query:   "UPDATE\n\tfoo",
-			expName: "UPDATE",
+			name:         "Newline and tab",
+			query:        "UPDATE\n\tfoo",
+			spanNameFunc: nil,
+			expName:      "UPDATE",
 		},
 		{
-			name:    "Additional whitespace",
-			query:   " \n SELECT\n\t   *   FROM users  ",
-			expName: "SELECT",
+			name:         "Additional whitespace",
+			query:        " \n SELECT\n\t   *   FROM users  ",
+			spanNameFunc: nil,
+			expName:      "SELECT",
 		},
 		{
-			name:    "Whitespace-only query",
-			query:   " \n\t",
-			expName: "UNKNOWN",
+			name:         "Whitespace-only query",
+			query:        " \n\t",
+			spanNameFunc: nil,
+			expName:      "UNKNOWN",
 		},
 		{
-			name:    "Empty query",
-			query:   "",
-			expName: "UNKNOWN",
+			name:         "Empty query",
+			query:        "",
+			spanNameFunc: nil,
+			expName:      "UNKNOWN",
+		},
+		{
+			name:         "Custom SQL name query",
+			query:        "-- name: GetUsers :many\nSELECT * FROM users",
```

Comment:
Could we add some more test cases here? 
1. Happy path with `/*` prefix
2. Happy path with `#` prefix
3. Some sad paths: like:
  * `--\nSELECT`
  * `#\nSELECT`
  * `/*\nSELECT`
  * `foo\nSELECT`
  * `-- foo\nSELECT`
  * ...